cmd/agent/android/README.md

```diff
@@ -0,0 +1,39 @@
+# Datadog Agent 6 on Android
+
```

Comment:
we need to mention that this is in `beta`, and that at the moment the Agent is not officially supported on Android

Reply:
done